openmdao/visualization/graph_viewer.py

```diff
@@ -590,6 +590,9 @@ def write_graph(G, prog='dot', display=True, outfile='graph.svg'):
     except AttributeError:
         raise AttributeError(f"pydot graph has no 'create_{ext}' method.")
 
+    if ext == 'svg':
+        outfile = outfile.replace('.svg', '.html')
+
```

Comment:
this might be a little confusing.. since the output file is different than what was requested..  
would it be better if the default extension is html, which uses `create-svg` internally?